backend/src/main/java/gov/cdc/usds/simplereport/service/idverification/LiveExperianService.java

```diff
@@ -0,0 +1,78 @@
+package gov.cdc.usds.simplereport.service.idverification;
+
+import static gov.cdc.usds.simplereport.service.idverification.ExperianTranslator.createInitialRequestBody;
+
+import gov.cdc.usds.simplereport.api.model.accountrequest.IdentityVerificationRequest;
+import gov.cdc.usds.simplereport.config.BeanProfiles;
+import gov.cdc.usds.simplereport.properties.ExperianProperties;
+import java.util.UUID;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+@Profile("!" + BeanProfiles.NO_EXPERIAN)
+public class LiveExperianService implements ExperianService {
+
+  private final ExperianProperties _experianProperties;
+  private RestTemplate _restTemplate;
+
+  @Autowired
+  public LiveExperianService(final ExperianProperties experianProperties) {
```

Comment:
I think this properties class needs to be included in `SimpleReportApplication.java` for the autowiring to work.

Reply:
Good call, just added it.